README.md

```diff
@@ -8,29 +8,12 @@ You’ll need an [EKS](https://aws.amazon.com/eks/) cluster to run against.
 
 ### Running on the EKS cluster
 1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) `kyverno-aws-adapter-sa` in your desired namespace (configured in `values.yaml`) and specified the role's ARN in the `roleArn` field inside `values.yaml` file.
```

Comment:
> specified the role's ARN in the `roleArn` field inside `values.yaml` file.

Where do I get this `values.yaml` file from? 🤔

Reply:
Yes. The documentation needs to be changed. To specify the roleArn at helm install time.